.github/workflows/ci.yml

```diff
@@ -36,6 +36,16 @@ jobs:
         sudo apt-get install -y \
           libbpf-dev \
           protobuf-compiler
+    - shell: python
+      id: args
+      run: |
+        import os
+
+        args='${{ matrix.args }}'.replace('-', '').split()
+        args='-'.join(args)
```

Comment:
This looks a bit weird (replacing hyphens and then joining with hyphens), might be worth a comment to describe why we're doing this + an example.

Also, in the case of `clippy -- -D warnings` do we end up with `clippy--D-warnings` or `clippy-D-warnings` ?